Automattic/jetpack
# Migrating the tracks-callables.js file to the Tracking package.

Related: 
https://github.com/Automattic/jetpack/issues/15929
https://github.com/Automattic/jetpack/pull/18869




Fixes #

#### Changes proposed in this Pull Request:

* Migrates the tracks-callables.js file to the Tracking package.

#### Does this pull request change what data or activity we track or use?
No Changes

#### Testing instructions:






* Install and activate a 2nd plugin that uses the Jetpack Connection.
* Go to the plugins page on your site
* Verify that w.js and tracks-callables.js are loaded
* Verify the deactivate dialog is presented asking to Deactivate or Disconnect and Deactivate when you deactivate Jetpack.
